src/vcr_clj/core.clj

```diff
@@ -81,26 +84,43 @@
                                    {:function var-name
                                     :arg-key arg-key}))))))))
 
+(defn record-new-episodes? [specs the-var-name]
+  (->> specs
+       (filter #(= (var-name (:var %)) the-var-name))
+       first
+       :record-new-episodes?))
+
 ;; Assuming that order is only preserved for calls to any var in
 ;; particular, not necessarily all the vars considered together.
 (defn playback
   [specs cassette func]
-  (let [the-playbacker (playbacker cassette :key)
+  (let [updated-cassette (atom cassette)
+        record! #(swap! updated-cassette update-in [:calls] conj %)
+        has-recording? #(get (indexed-cassette cassette) [%1 %2])
```

Comment:
this calls `indexed-cassette` on each call, which is unnecessary work since it's being called with the same argument each time